api/Render/AddHotRender.php

```diff
@@ -0,0 +1,71 @@
+<?php declare(strict_types=1);
+
+
+namespace HomeCEU\DTS\Api\Render;
+
+
+use HomeCEU\DTS\Api\ApiHelper;
+use HomeCEU\DTS\Persistence\CompiledTemplatePersistence;
+use HomeCEU\DTS\Persistence\HotRenderPersistence;
+use HomeCEU\DTS\Persistence\TemplatePersistence;
+use HomeCEU\DTS\Render\CompilationException;
+use HomeCEU\DTS\Repository\HotRenderRepository;
+use HomeCEU\DTS\Repository\TemplateRepository;
+use HomeCEU\DTS\UseCase\AddHotRender as AddHotRenderUseCase;
+use HomeCEU\DTS\UseCase\AddHotRenderRequest;
+use HomeCEU\DTS\UseCase\InvalidHotRenderRequestException;
+use Psr\Container\ContainerInterface;
+use Psr\Http\Message\RequestInterface as Request;
+use Psr\Http\Message\ResponseInterface as Response;
+
+class AddHotRender {
+  /** @var \HomeCEU\DTS\UseCase\AddHotRender */
+  private $useCase;
+
+  public function __construct(ContainerInterface $container) {
+    $conn = $container->dbConnection;
```

Comment:
if you only need the dbConnection, why don't you just pass that instead of the container?

Reply:
Slim actually builds this class in routes.php and injects the container. I'm just working with what Slim gives me rather than to create my own workaround.